web/client/src/library/components/tasksProgress/TasksProgress.tsx

```diff
@@ -18,7 +18,7 @@ interface PropsTasks {
   showProgress?: boolean
   showVirtualUpdate?: boolean
   planState: PlanState
-  setRefTaskProgress: RefObject<HTMLDivElement>
+  setRefTaskProgress?: RefObject<HTMLDivElement>
```

Comment:
What does this do? What happens if it's null or undefined?

Reply:
When applying a plan and backfill starts , the UI scrolls to TasksProgress to highlight to a user that it started but for that to work we need actual HTML element of  the `TasksProgress` component . if `setRefTaskProgress` is `null` it just wont set that reference and scrolling to TasksProgress wont happen